libs/examples/src/chart/chart-bar-example/chart-bar-example.ts

```diff
@@ -13,14 +13,53 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {
+  DtChart,
+  DtChartTooltipConfig,
+  DtChartTooltipData,
+  DT_CHART_TOOLTIP_CONFIG,
+} from '../../../../barista-components/chart';
+import { PlotBackgroundInfo } from '../../../../barista-components/chart/src/utils';
+import { Component } from '@angular/core';
 
-// tslint:disable:no-magic-numbers
+const getTooltipPosition = (
+  data: DtChartTooltipData,
+  chart: DtChart,
+  plotBackgroundInfo?: PlotBackgroundInfo,
+): { x: number; y: number } => {
+  const containerElement: HTMLElement = chart._container.nativeElement;
+  const containerElementBB = containerElement.getBoundingClientRect();
+  const { x, y } = getVerticalScrollPositioner(
+    data,
+    plotBackgroundInfo as PlotBackgroundInfo,
+  );
 
-import { Component } from '@angular/core';
+  return {
+    x: containerElementBB.left + x,
+    y: containerElementBB.top + y,
+  };
+};
+
+const getVerticalScrollPositioner = (
```

Comment:
NIT: The naming of this function is not ideal - it has nothing to do with scrolling - maybe `verticalPositionFunction` is a better fit.